test_project/testapp/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+class Team(models.Model):
+    name = models.CharField(max_length=20)
+    def __str__(self):
+        return self.name
+class Player(models.Model):
+    name = models.CharField(max_length=20)
+    current_team = models.ForeignKey("Team",related_name="current_players",help_text="This demonstrate the wrapper adding a create button only")
```

Comment:
Your Python formatting is seriously broken in this file ;) Try `pylint` or `pep8`